plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMaterializedView.java

```diff
@@ -55,6 +88,29 @@ protected DistributedQueryRunner createQueryRunner()
                     .setCatalog("iceberg2")
                     .build();
 
+            RandomTableFunctionProcessorProvider processorProvider = new RandomTableFunctionProcessorProvider();
```

Comment:
inline

Reply:
I can't i want to always use the same instance of processor